pkg/xray/resource_xray_workers_count.go

```diff
@@ -0,0 +1,280 @@
+package xray
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-shared/util"
+)
+
+func resourceXrayWorkersCount() *schema.Resource {
+	newContentSchema := map[string]*schema.Schema{
+		"new_content": {
+			Type:        schema.TypeInt,
+			Required:    true,
+			Description: "Number of workers for new content",
+		},
+	}
+
+	newExistingContentSchema := util.MergeMaps(
```

Comment:
I don't get why you do this merging - is it somehow shared?

Reply:
Yep, `newContentSchema` is used for `impact_analysis` and `notification` attributes. Separate from all the other attributes.